holidayextras/react-date-time-range
# PROD-1247 updating react-date-time-range to react v0.14

#### What does this PR do? (please provide any background)

This PR updates the react-date-time-range to react v0.14+
#### What tests does this PR have?

This PR has unit tests. use npm test
#### How can this be tested?

run npm start go to the localhost link it provides and make sure that it works.
http://localhost:3000/example.html
#### Screenshots / Screencast

http://grab.by/NHfo
#### What gif best describes how you feel about this work?

![](http://tclhost.com/rAFv3CN.gif)

---
#### Reviewers

**Review 1**

**Review 2** *

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
